sdk/eventhub/Azure.Messaging.EventHubs/CHANGELOG.md

```diff
@@ -1,6 +1,55 @@
 # Release History
 
-## 5.0.0-preview.5 (2019-11-04)
+## 5.0.0-preview.6
+
+### Acknowledgements
+
+Thank you to our developer community members who helped to make the Event Hubs client libraries better with their contributions to this release:
+
+- Alberto De Natale _([GitHub](https://github.com/albertodenatale))_
+
+### Changes
+
+#### Bug fixes and foundation improvements
+
+- A bug with the use of Azure.Identity credential scopes has been fixed; Azure identities should now allow for proper authorization with Event Hubs resources.   
+_(A community contribution, courtesy of [albertodenatale](https://github.com/albertodenatale))_
+
+- A bug with the renewal of connection string-based credentials has been fixed; clients using a connection string will now properly refresh when necessary instead of experiencing an error.
+
+- Some performance tuning has been done, focusing on reducing allocations associated with frequently used types where possible.
+
+#### Consuming events
+
+- The `EventHubsConsumerClient` is no longer bound to a partition, an owner level, or tracking of last enqueued event properties; these attributes may be specified when requesting to read events, allowing more granular control over behavior without the need to create multiple clients.
+
+- Events may now be read across all partitions of an Event Hub using the `ReadEvents` method of the consumer client.  This is intened for exploring Event Hubs and not recommended for production use; for production scenarios, please consider using the `EventProcessorClient` as a more robust alternative.
+
+- Events read using the consumer client are now accompanied by a `PartitionContext` helping to identify the source partiiton for an event and allowing for partition-specific operations, such as reading the last enqueued event properties for that partition.
```

Comment:
```suggestion
- Events read using the consumer client are now accompanied by a `PartitionContext` helping to identify the source partition for an event and allowing for partition-specific operations, such as reading the last enqueued event properties for that partition.
```

Reply:
Fixed this as part of earlier feedback.  Thanks!